R/markdown.R

```diff
@@ -128,3 +141,31 @@ mdxml_image = function(xml) {
   title <- xml_attr(xml, "title")
   paste0("\\figure{", dest, "}{", title, "}")
 }
+
+mdxml_heading <- function(xml, state) {
+  if (state$tag$tag != "@includeRmd") {
+    return(mdxml_unsupported(xml, state$tag, "markdown headings"))
```

Comment:
Lets get this PR merged and then add heading support everywhere.

Reply:
OK.